### ISSUE TITLE
Adding maps it's nearly impossible

### ISSUE THREAD

**Issue by DiomedesDominguez** [SuavePirate/SuaveControls.MaterialFormControls#21]
I'm getting the error message: ```Default constructor not found for type SuaveControls.MaterialForms.BorderlessEntry``` just right after adding a new XF ContentPage with a Xamarin.Forms.Maps.

I'm using VS for Mac Enterprise v7.3.3 build 17 on MacOSX High Sierra.

I'll try in Windows to see if this is something about the OS.

**Comment by SuavePirate** [comment]
What version of Xamarin.Forms?

**Comment by SuavePirate** [comment]
Does it work without maps at all? And did you call to initialize the renderer like the readme mentions?

```
BorderlessEntry.Init();
```

**Comment by DiomedesDominguez** [comment]
Yes, before I added the ```Xamarin.Forms.Maps``` nugget package and initialize it.
